packages/embeds/embed-core/playwright/lib/testUtils.ts

```diff
@@ -104,10 +116,35 @@ export async function bookFirstEvent(username: string, frame: Frame, page: Page)
   await frame.fill('[name="email"]', "embed-user@example.com");
   await frame.press('[name="email"]', "Enter");
   const response = await page.waitForResponse("**/api/book/event");
-  const responseObj = await response.json();
-  const bookingId = responseObj.uid;
+  const booking = (await response.json()) as { uid: string; eventSlug: string };
+  booking.eventSlug = eventSlug;
```

Comment:
Useful for reschedule test to know, which `eventSlug` would `reschedule/{UID}` would get redirected to.